src/components/templates/appLayout.tsx

```diff
@@ -48,16 +49,16 @@ export const AppLayout = ({ className, hideTopbar }: { className?: string; hideT
 
 	return (
 		<div className={appLayoutClasses}>
-			<Sidebar />
+			{!isTable ? <Sidebar /> : null}
```

Comment:
if we don't load and display `<Sidebar />`, would the projectList load?

Reply:
if we don't load and display `<Sidebar />`, projectList don't load

Reply:
On mobile, we would like to show the menu (the sidebar).
Not on the intro/welcome page, but on any other page we will need it.